references.bib

```diff
@@ -0,0 +1,278 @@
+%used
+@inproceedings{botea2013pathfinding,
+  title={Pathfinding in games},
+  author={Botea, Adi and Bouzy, Bruno and Buro, Michael and Bauckhage, Christian and Nau, Dana},
+  booktitle={Dagstuhl Follow-Ups},
+  volume={6},
+  year={2013},
+  organization={Schloss Dagstuhl-Leibniz-Zentrum fuer Informatik}
+}
+%used
+@article{botea2004near,
+  title={Near optimal hierarchical path-finding},
+  author={Botea, Adi and M{\"u}ller, Martin and Schaeffer, Jonathan},
+  journal={Journal of game development},
+  volume={1},
+  number={1},
+  pages={7--28},
+  year={2004},
+  publisher={Citeseer}
+}
+%used
+@inproceedings{botea2011ultra,
+  title={Ultra-Fast Optimal Pathfinding without Runtime Search.},
+  author={Botea, Adi and others},
+  booktitle={AIIDE},
+  year={2011}
+}
+%used
+@inproceedings{cazenave2006optimizations,
+  title={Optimizations of data structures, heuristics and algorithms for path-finding on maps},
+  author={Cazenave, Tristan},
+  booktitle={Computational Intelligence and Games, 2006 IEEE Symposium on},
+  pages={27--33},
+  year={2006},
+  organization={IEEE}
+}
+%used
+@article{cui2011based,
+  title={A*-based pathfinding in modern computer games},
+  author={Cui, Xiao and Shi, Hao},
+  journal={International Journal of Computer Science and Network Security},
+  volume={11},
+  number={1},
+  pages={125--130},
+  year={2011},
+  publisher={International Journal of Computer Science and Network Security (IJCSNS)}
+}
+%used
+@article{cui2011direction,
+  title={Direction oriented pathfinding in video games},
+  author={Cui, Xiao and Shi, Hao},
+  journal={International Journal of Artificial Intelligence \& Applications},
+  volume={2},
+  number={4},
+  pages={1},
+  year={2011},
+  publisher={Academy \& Industry Research Collaboration Center (AIRCC)}
+}
+%used
+@article{algfoor2015comprehensive,
+  title={A comprehensive study on pathfinding techniques for robotics and video games},
+  author={Algfoor, Zeyad Abd and Sunar, Mohd Shahrizal and Kolivand, Hoshang},
+  journal={International Journal of Computer Games Technology},
+  volume={2015},
+  pages={7},
+  year={2015},
+  publisher={Hindawi Publishing Corp.}
+}
+%used
+@inproceedings{kallmann2010navigation,
+  title={Navigation queries from triangular meshes},
+  author={Kallmann, Marcelo},
+  booktitle={International Conference on Motion in Games},
+  pages={230--241},
+  year={2010},
+  organization={Springer}
+}
+%used
+@article{korf1990real,
+  title={Real-time heuristic search},
+  author={Korf, Richard E},
+  journal={Artificial intelligence},
+  volume={42},
+  number={2-3},
+  pages={189--211},
+  year={1990},
+  publisher={Elsevier}
+}
+%used
+@article{sturtevant2007memory,
+  title={Memory-Efficient Abstractions for Pathfinding.},
+  author={Sturtevant, Nathan R},
+  journal={AIIDE},
```

Comment:
AIIDE is a conference, not a journal -- this should be an `@inproceedings`. Also you should spell it out as "AAAI Conference on Artificial Intelligence and Interactive Digital Entertainment (AIIDE)". There are several other references from AIIDE which need fixing too.